### PR TITLE
Add support of bucket sync run command

### PR DESCRIPTION
**Pull request by udaysk23**
Add support of bucket sync run command

Signed-off-by: uday kurundwade <ukurundw@redhat.com>

### CODE REVIEW

**Comment by udaysk23** [comment]
Following is the log link of successful run
http://pastebin.test.redhat.com/1052163

**Comment by udaysk23** [comment]
Following is the log link of successful run
http://pastebin.test.redhat.com/1052163

File: rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py [hunk 1]
```diff
@@ -332,7 +332,16 @@ def test_exec(config):
...
-                    if config.bucket_sync_crash is True:
+                    if config.bucket_sync_run:
+                        _, source_zone = utils.get_realm_source_zone_info()
+                        log.info(f"Source zone name: {source_zone}")
+                        cmd = f"radosgw-admin bucket sync run --bucket={bucket.name} --source-zone={source_zone}"
+                        out = utils.exec_shell_cmd(cmd)
+                        if out is False:
+                            raise TestExecError(
+                                "Command is throwing error while running bucket sync run"
+                            )
```
Comment (psathyan) [inline rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py:343]: There are too many `if statements` within this method making it monolithic. IMHO, we can write smaller methods having a specific purpose. For example
```
def check_bucket_syn(config, bucket):
  if not config.bucket_sync_run: return

  log.info("Checking if buckets are syncrhonizing"
  _, source_zone = utils.get_realm_source_zone_info()
  log.debug(f"Source zone name: {source_zone}")
  ...
```

**Review by anrao19 — DISMISSED** [review]
LGTM

**Review by ckulal — APPROVED** [review]

**Review by psathyan — APPROVED** [review]

**Pull request merged**